test-support/factories/citation.js

```diff
@@ -0,0 +1,8 @@
+import FactoryGuy from 'ember-data-factory-guy';
+import faker from 'faker';
+
+FactoryGuy.define('citation', {
+    default: {
+        citation: faker.lorem.words(10)
```

Comment:
Remind me, if you please: does this set up a single value for every instance of the factory? I notice that in many of our other factories, we provide a function that can be called per-instance instead:

`() => faker.lorem.words(10)`

Noted for this one field, but if necessary we should review all fields in the PR that use `faker` and treat them consistently.